cmd/rollkit/main.go

```diff
@@ -11,6 +11,10 @@ import (
 )
 
 func main() {
+	if err := cmd.InterceptCommand(); err == nil {
```

Comment:
I would add a comment here explaining this more to help our future selves and new developers understand what this is doing and why we have it.